### ISSUE TITLE
"connect" command of msf console seems to be malfunctional - no output.

### ISSUE THREAD

**Issue by Mori-mori-mori** [rapid7/metasploit-framework#5074]
In case of simple command string like "connect ip_addr some_port" it just returns a string like "Connected to ip_addr:some_port" and nothing more, no repsonse/banner from http or smtp server on this port, nothing. But if you issue the "help" command after that, it actually print the output of this simple connect command, without printing the help contents; Only the second "help" command will print the help contents.
In case you try somewhat more complex connect string. like "connect -i /some/file/with/http/request.txt ip_addr some_port", all looks the same, but now even this trick with subsequent "help" command won't work, not output at all, "help" print help content at first attempt.

**Comment by rwhitcroft** [comment]
Works fine here:
```
msf > connect localhost 22
[*] Connected to localhost:22
SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2

Protocol mismatch.
msf >
```

If you telnet to the host/port, do you see a banner? When you say `help` do you mean `connect -h`? The `help` command has nothing to do with the `connect` command.

**Comment by Mori-mori-mori** [comment]
I'm sorry, I've got a wrong impression on how it should work. It seems that upon typing enter it connects to specified port and will await further commands typed in. So when I typed "help" after that (thinking it just hanged, or nothing happened), it actually sent it to http server, resulting in response from it ("Bad request").
But I still haven't been able to understan why "connect -i /some/file/with/http/request.txt ip_addr some_port". It doesn't produce any errors, and it returns to msfcli's prompt after execution, but no output at all, aside from "Connected to ip_addr:some_port". When I try to execute "cat/some/file/with/http/request.txt | nc ip_addr some_port" (with the file of the same contents), it successfuly produce respose from the server.
